tests/api_validation_test.go

```diff
@@ -97,9 +97,9 @@ var _ = Describe("Validation tests", func() {
 					return false
 				}, timeout, pollingInterval).Should(BeTrue())
 			},
-				table.Entry("fail with zero PVC size", "0"),
-				table.Entry("fail with negative PVC size", "-500m"),
-				table.Entry("fail with invalid PVC size", "invalid_size"),
+				table.Entry("[test_id:1033]fail with zero PVC size", "0"),
```

Comment:
for those cases, I have old case CNV-1027 to cover, are we sure we gonna use a new id instead ?